src/components/WalletDropdown/MiniPortfolio/Activity/types.ts

```diff
@@ -12,7 +12,7 @@ export type Activity = {
   title: string
   descriptor?: string
   logos?: Array<string | undefined>
-  currencies?: Array<Currency>
+  currencies?: Array<Currency | undefined>
```

Comment:
These can be narrowed:

```
  logos?: [string | undefined, string | undefined],
  currencies?: [Currency | undefined, Currency | undefined],
```

Reply:
these can have any number of items depending on the tx type - i'd rather not make that big of a type change in this PR